src/xcode/ENA/ENA/Resources/Localization/en.lproj/privacy-policy.html

```diff
@@ -472,15 +476,12 @@ <h3>
 			</p>
 		</p>
 		<p>
-			For this purpose, the app runs in the background and retrieves an
-			up-to-date list every day of random IDs, and possible information about the
-			onset of symptoms, from the server system. This list contains the random
-			IDs and possible symptom information of users who have tested positive for
-			coronavirus and voluntarily used the warning feature in their app, which is
-			the official coronavirus app in any country participating in the
-		transnational warning system (see Section 7) (hereinafter referred to as a    <strong>positive list</strong>). The random IDs in the positive list also
-			contain a transmission risk value and an indication of the type of
-			diagnosis (see Section 6 c.).
+			For this purpose, the app retrieves an up-to-date list from the server system several times a day. 
+			This list contains the random IDs, along with any voluntary symptom information, of users who have 
```

Comment:
"have tested" or "have been tested"?

Reply:
Hi Andreas 'have tested' is fine here. This is also a translation provided by the law firm's translators.